include/nuttx/compiler.h

```diff
@@ -944,6 +944,10 @@
 
 #endif
 
+#ifndef CONFIG_HAVE_LONG_LONG
+#  undef CONFIG_FS_LARGEFILE
+#endif
+
```

Comment:
Why not
```
config FS_LARGEFILE
	bool "Large File Support"
	default n
	depends on HAVE_LONG_LONG
```
in Kconfig instead?

Reply:
Can not since HAVE_LONG_LONG isn't a Kconfig option, but define in:
https://github.com/apache/nuttx/blob/master/include/nuttx/compiler.h#L283